src/android/io/cozy/plugins/listlibraryitems/ListLibraryItems.java

```diff
@@ -271,6 +284,36 @@ private static String getMimeType(String url) {
         return type;
     }
 
+    // This method parse a ISO8601 (non http date format) to a Date
+    public static Date parseIso8601DateTime(String string) {
+        if (string == null) {
+            return null;
+        }
+        String s = string.replace("Z", "+00:00");
+        for (String pattern : SUPPORTED_ISO_8601_PATTERNS) {
+            String str = s;
+            int colonPosition = pattern.lastIndexOf('Z') - TICK_MARK_COUNT + COLON_PREFIX_COUNT;
+            if (str.length() > colonPosition) {
+                str = str.substring(0, colonPosition) + str.substring(colonPosition + 1);
+            }
+            try {
+                return new SimpleDateFormat(pattern).parse(str);
+            } catch (final ParseException e) {
+                // try the next one
+            }
+        }
+        return null;
+    }
+
+    // This method format a Date to a RFC 1123 String
+    public static String formatRfc1123DateTime(Date date, TimeZone timeZone) {
+        SimpleDateFormat dateFormat = new SimpleDateFormat(RFC_1123_DATE_TIME);
+        if (timeZone != null) {
+            dateFormat.setTimeZone(timeZone);
+        }
+        return dateFormat.format(date);
+    }
```

Comment:
can we declare the methods relative to DateManipulation outside of the ListItem class?

Reply:
Sure, but is it necessary to create a new file, or leave everything on this one ?

Reply:
Having it in the same file is enough for now :).